examples/src/main/java/org/apache/spark/examples/JavaLogQuery.java

```diff
@@ -40,7 +38,7 @@
  */
 public final class JavaLogQuery {
 
-  public static final List<String> exampleApacheLogs = Lists.newArrayList(
+  public static final List<String> exampleApacheLogs = Arrays.asList(
```

Comment:
Although this isn't specific to Java 8, the existence of Arrays.asList should mean we don't need Guava's Lists.newArrayList, and, we should prefer examples that don't involve third party libraries.